doc/config.md

```diff
@@ -1060,6 +1060,20 @@ they are logged but ignored otherwise.
 Set the pool mode specific to this database. If not set,
 the default `pool_mode` is used.
 
+### host_strategy
```

Comment:
I think it would make sense to change the name of the option to `load_balance_hosts`. This option got added to libpq for PG16 (by me). It seems nice to use consistently named options across projects. The `last_successful` option name should then be changed to `disable`. And lets change `round_robin` to `round-robin`, to be consistent with other arguments like `verify-full` and `read-write`.

PS. I especially think consistency is nice here, because you mentioned you also want to add `target_session_attrs` to PgBouncer. Since `target_session_attrs` and `load_balance_hosts` can be used nicely together, I think it's especially useful to have the same names for both Postgres and PgBouncer.

Reply:
Btw, adding a `random` as an option would be a nice future improvement to make sure PgBouncer supports the same as libpq. But that should definitely be another follow up PR. We can merge this PR easily without the random functionality.

Reply:
> I think it would make sense to change the name of the option to load_balance_hosts. This option got added to libpq for PG16 (by me).

Ha, good call. I noticed that you contributed this feature in the release notes. +100 to consistent naming with libpq. I'll take care of that.

Reply:
I prefer the initial naming. It seems clear. load_balance_hosts seems to be a misnomer, since no "load" and no "balancing" is involved?

Reply:
> I prefer the initial naming. It seems clear.

I think the initial name was a fine name too. But I think being in line with naming of libpq, jdbc and npgsql is a big advantage of the load_balance_hosts naming.

> load_balance_hosts seems to be a misnomer, since no "load" and no "balancing" is involved?

The "load" is the number of connection that are made by pgbouncer to the database, and those connections are "balanced" across the different servers in a round-robin fashion (or random fashion when implemented in the future). And you can enable this balancing by using `disable` which will select a working host and send all traffic there. 

IMHO, the old naming describes how pgbouncer chooses hosts. The new naming describes what the user will see happening to their setup by changing the setting. I think I personally like names that describe the effects instead of the method.